tests/entropy_adiabat/screen-output

```diff
@@ -0,0 +1,555 @@
+   Copying properties into prescribed compositional field density_field.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 2+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 0, 8.56783e-05, 0, 0.000386971
+      Relative nonlinear residual (total system) after nonlinear iteration 1: 0.000386971
+
+   Copying properties into prescribed temperature field.
+   Solving entropy system ... 4 iterations.
+   Copying properties into prescribed compositional field density_field.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 2+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 0, 1.68331e-07, 0, 3.91883e-05
+      Relative nonlinear residual (total system) after nonlinear iteration 2: 3.91883e-05
+
+   Copying properties into prescribed temperature field.
+   Solving entropy system ... 3 iterations.
+   Copying properties into prescribed compositional field density_field.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 2+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 0, 2.64371e-09, 0, 5.10681e-06
+      Relative nonlinear residual (total system) after nonlinear iteration 3: 5.10681e-06
+
+
+   Postprocessing:
+     Writing graphical output:           output-entropy_adiabat/solution/solution-00015
+     RMS, max velocity:                  0.00874 m/year, 0.00995 m/year
+     Temperature min/avg/max:            1491 K, 1623 K, 1742 K
+     Mass fluxes through boundary parts: 0 kg/yr, 0 kg/yr, 4.403e+05 kg/yr, -4.372e+05 kg/yr
+     Compositions min/max/mass:          2357/2534/4.013e+13 // 3363/4397/6.614e+13
+
+*** Timestep 16:  t=1e+07 years
+   Copying properties into prescribed temperature field.
+   Solving entropy system ... 5 iterations.
+   Copying properties into prescribed compositional field density_field.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 2+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 0, 2.28585e-05, 0, 4.82553e-05
+      Relative nonlinear residual (total system) after nonlinear iteration 1: 4.82553e-05
+
+   Copying properties into prescribed temperature field.
+   Solving entropy system ... 3 iterations.
+   Copying properties into prescribed compositional field density_field.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 2+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 0, 7.91949e-09, 0, 2.24843e-06
+      Relative nonlinear residual (total system) after nonlinear iteration 2: 2.24843e-06
+
+
+   Postprocessing:
+     Writing graphical output:           output-entropy_adiabat/solution/solution-00016
+     RMS, max velocity:                  0.00874 m/year, 0.00995 m/year
+     Temperature min/avg/max:            1492 K, 1624 K, 1742 K
```

Comment:
Not something to do right now, but should there now be an entropy postprocessor?